src/backend/distributed/shared_library_init.c

```diff
@@ -1414,7 +1414,7 @@ RegisterCitusConfigVariables(void)
 					 "and operating system name. This configuration value controls "
 					 "whether these reports are sent."),
 		&EnableStatisticsCollection,
-#if defined(HAVE_LIBCURL) && defined(ENABLE_CITUS_STATISTICS_COLLECTION)
+#if defined(HAVE_LIBCURL)
```

Comment:
why did we remove the ENABLE_CITUS_STATISTICS_COLLECTION check? 

when I compile Citus, I now see HTTP errors because the statistics collection started talking to a non-existent endpoint